src/core/Board.js

```diff
@@ -51,11 +64,16 @@ export const Board = () => {
   };
 
   return (
-    <>
-      <p>
-        Tries: {turnCounter}, Matches: {matchCounter}
-      </p>
-      <div className={styles.board}>
+    <div className={styles.board}>
```

Comment:
If I understand, we either show the board or the win screen, but your code renders both at the same time. You could use conditional rendering (read: if-statement) here to clarify the logic :)

This approach will also prevent bugs where the state of the win screen doesn't really work for the board state. If a component is rendered, it's still "doing stuff".

Reply:
Ooh!

Reply:
I'm rendering the modal (empty though) in both states so that I can animate it fading in, but I'm not rendering the board at all when the winner modal is open anymore. This was a nice little "aha" moment. Thanks!